Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -604,6 +604,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MARKETING_VERSION = 8.1.2;
```

Comment:
Our release script will update this to 9.0.0 when we do the release, so I set it back to 8.1.2 for the time being.